src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -50,6 +51,33 @@ public function load(string $path, string ...$paths): void
         $this->doLoad(false, $path, $paths);
     }
 
+    /**
+     * Loads one or several .env and the corresponding env.local, env.$env and env.$env.local files if they exist.
+     *
+     * .env.local is always ignored in test env because tests should produce the same results for everyone.
```

Comment:
A developer may have to customize the `DATABASE_URL` for tests execution on his local machine. If `.env.test.local` is ignored, how the database should be configured?

Reply:
`.env.test.local` is **not** ignored. Only `.env.local` is.

Reply:
Read too fast. So it's fine :)